### PR TITLE
Add Copy Icon next to address in Twin page.

### PR DESCRIPTION
**Pull request by AlaaElattar**
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1147
Screenshot from 2023-10-09 16-27-25

Screenshot from 2023-10-09 16-27-30

### CODE REVIEW

**Comment by AlaaElattar** [comment]
- Update size of Icon 

image

File: packages/playground/src/portal/twin_view.vue [hunk 1]
```diff
@@ -36,7 +36,9 @@
               <v-divider></v-divider>
               <v-list-item>
                 {{ profileManager.profile?.address }}
-                <v-icon style="padding-left: 3rem" @click="copy(profileManager.profile?.address as string)"
+                <v-icon
```
Comment (zaelgohary) [inline packages/playground/src/portal/twin_view.vue:39]: Can you make the icon at the end of the row instead?

Comment (AlaaElattar) [reply packages/playground/src/portal/twin_view.vue]: image

**Review by amiraabouhadid — APPROVED** [review]
looks good!
image
Screenshot from 2023-10-10 12-12-45

**Review by zaelgohary — APPROVED** [review]

File: packages/playground/src/portal/twin_view.vue [hunk 2]
```diff
@@ -96,6 +103,18 @@ async function UpdateRelay() {
     console.log("could not update relay or pk, Error: ", e);
   }
 }
+
+function copy(id: string) {
+  navigator.clipboard.writeText(id);
+  createToast(`Address copied to clipboard`, {
```
Comment (AhmedHanafy725) [inline packages/playground/src/portal/twin_view.vue:109]: please use the custom toast method

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**